software/devices/dmx/dmx.cpp

```diff
@@ -1,48 +1,79 @@
+#include <SparkFunDMX.h>
+
 #include "../../arduino/RadioHeadRadio.hpp"
+#include "../../generic/NetworkManager.hpp"
+
+#define DMX_CHANNELS 192
```

Comment:
You should use `const`s instead of defines.

Side rant: I hate that example code still uses `#defines` as if compilers were still stuck in the 90s and couldn't optimize out consts.